crates/control/src/motion/joint_command_sender.rs

```diff
@@ -14,34 +11,16 @@ pub struct CreationContext {}
 
 #[context]
 pub struct CycleContext {
-    pub positions: AdditionalOutput<Joints<f32>, "positions">,
-    pub compensated_positions: AdditionalOutput<Joints<f32>, "compensated_positions">,
-    pub positions_difference: AdditionalOutput<Joints<f32>, "positions_difference">,
+    pub final_positions: AdditionalOutput<Joints<f32>, "final_positions">,
```

Comment:
Could you improve "final" here? It is not consistent with the other naming style. We don't call things "first", "after first", "one before final", "final" because it does not give much information how it differs from the other ones. What is the difference between the other ones? Then prefix it with that.

Reply:
Done. Renamed to executed_positions

Reply:
Ops forgot to delete the file, isn't used anymore because there exists JointsCommand<T>

Reply:
Yea, exposing the `JointsCommand` directly makes more sense. Done